README.rst

```diff
@@ -424,6 +428,82 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
      - ``lowest``
      - Decides on what basis the Karapace cluster master is chosen (only relevant in a multi node setup)
 
+Karapace Schema Registry authorization file is an optional JSON configuration, which contains a list of authorized users in ``users`` list and a set of access control rules in ``permissions`` list.  Karapace uses HTTP Basic Authentication for user authentication.
+
+Each user entry contains following attributes:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``username``
+     - A string
+   * - ``algorithm``
+     - One of supported hashing algorithms, ``scrypt``, ``sha1``, ``sha256``, or ``sha512``
+   * - ``salt``
+     - Salt used for hashing the password
```

Comment:
How exactly is it used?

Reply:
Do we really need to provide the details info? I'd probably just post a link to https://en.wikipedia.org/wiki/Salt_(cryptography)